Kernel/linker_amd64.ld

```diff
@@ -4,11 +4,11 @@ KERNEL_VIRTUAL_BASE = 0x2000000000;
 
 PHDRS
 {
-  boot_bss PT_LOAD ;
-  text PT_LOAD ;
-  data PT_LOAD ;
-  bss PT_LOAD ;
-  ksyms PT_LOAD ;
+  super_pages PT_LOAD FLAGS((1 << 1) | (1 << 2));
```

Comment:
FLAGS feels like it could use a comment explaining that it's RWX just like normal file permissions. The bit shifting looks a bit cryptic. Almost debating whether FLAGS(6) would be easier to grok than FLAGS((1 << 1) | (1 << 2))

Reply:
Hm yeah, what if we preprocess this file with the C preprocessor and then just `#define` PF_X, PF_W and PF_R?

This would also allow us to use a macro for the base address - although that might be less of a concern once I get KASLR and relocations working (no work on that yet).

Reply:
I've updated the linker script to use the C preprocessor. :)